test/src/require.js

```diff
@@ -71,6 +89,32 @@ test('testing', (t) => {
   // function to call
   t.is('function', typeof (router.stack[3].route.stack[0].handle));
 
+  // MIDDLEWARES
+  t.is(undefined, router.stack[4].route.stack[1].path);
```

Comment:
what is this exactly? why do we test if something is undefined? is it not undefined if we use no middleware?

Reply:
Oops.